configs/dino/dino-5scale_swin-l_8xb2-12e_coco.py

```diff
@@ -0,0 +1,181 @@
+_base_ = [
+    '../_base_/datasets/coco_detection.py', '../_base_/default_runtime.py'
```

Comment:
Hi~, thanks for your great work!
I think the config is better to inherit from [dino-4scale_r50_8xb2-12e_coco.py](https://github.com/RistoranteRist/mmdetection/blob/dino-swin/configs/dino/dino-4scale_r50_8xb2-12e_coco.py).

Reply:
I cannot agree it more.